src/Microsoft.Dnx.Runtime.Sources/Impl/DependencyInjection/PlatformServicesExtension.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Dnx.Runtime.Common.DependencyInjection
+{
+    internal static class PlatformServicesExtension
+    {
+        public static void AddPlatformServices(this ServiceProvider serviceProvider)
```

Comment:
Add PlatformServices parameter and another overload to call it with PlatformServices.Default?